src/pages/News/News.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from "react";
+import { loadNews } from "../../api/news";
+
+export const News = () => {
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [news, setNews] = useState([]);
+
+  useEffect(async () => {
```

Comment:
TODO: move loader to separate func

const getNews = async () => {}

useEffect(getNews(), [])